descriptors/ModuleDescriptor-template.json

```diff
@@ -328,7 +328,9 @@
           "permissionsRequired": ["inventory.holdings.item.put"],
           "modulePermissions": [
             "inventory-storage.holdings.item.get",
-            "inventory-storage.holdings.item.put"
+            "inventory-storage.holdings.item.put",
+            "inventory-storage.holdings.collection.get",
+            "inventory-storage.holdings.batch.post"
```

Comment:
These 2 permissions are used by Holding Update Kafka consumer that needs to fetch other holdings that using in the same order and update them to use the same instanceId. In this case all holdings in order will have holdings connected to the same instance:
https://github.com/folio-org/mod-orders-storage/blob/c0923e3a120d7108241e0ecc1f7a22cb7da246ae/src/main/java/org/folio/services/inventory/InventoryUpdateService.java#L47